www/include/configuration/configNagios/DB-Func.php

```diff
@@ -219,6 +219,34 @@ function insertNagiosInDB()
     return ($nagios_id);
 }
 
+/**
+ * Calculate the sum of bitwise for a POST QuickForm array
+ *
+ * The array format
+ *
+ * array[key] => enable
+ *  Key int the bit
+ *  Enable 0|1 if the bit is activate
+ *
+ * if found the bit -1 (all) or 0 (none) activate return the value
+ *
+ * @param array $list The POST QuickForm table
+ * @return int The bitwise
+ */
+function calculateBitwise($list)
+{
+    $bitwise = 0;
```

Comment:
Should not be in lowerCamelCase (https://github.com/centreon/centreon/tree/master/doc/coding-style/php#php)

Reply:
bitwise is one word, so calculateBitwise is lowerCamelCase